README.md

```diff
@@ -1,39 +1,106 @@
+<div align="center">
+    <a href="https://github.com/CrayLabs/SmartSim"><img src="https://raw.githubusercontent.com/CrayLabs/SmartSim/master/doc/images/SmartSim_Large.png" width="90%"><img></a>
+    <br />
+    <br />
+    <div display="inline-block">
+        <a href="https://github.com/CrayLabs/SmartSim-Scaling"><b>Home</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://www.craylabs.org/docs/installation_instructions/basic.html"><b>Install</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://www.craylabs.org/docs/overview.html"><b>Documentation</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://join.slack.com/t/craylabs/shared_invite/zt-nw3ag5z5-5PS4tIXBfufu1bIvvr71UA"><b>Slack</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://github.com/CrayLabs"><b>Cray Labs</b></a>&nbsp;&nbsp;&nbsp;
+    </div>
+    <br />
+    <br />
+</div>
+----------
+
 # SmartSim Scaling
 
 This repository holds all of the scripts and materials for testing
-the scaling of SmartSim and the SmartRedis clients.
+the scaling of SmartSim and SmartRedis clients.
+
+The scalability tests mimic an HPC workload by making calls to SmartSim 
+and SmartRedis infrastructure to complete in parallel highly complex, data-intensive 
+tasks that are spread across compute resources. 
+These applications are used to test the performance of SmartSim and 
+SmartRedis across various system types.
+
+## Scalability Tests Supported
+
+The SmartSim-Scaling repo offers three scalability tests with 
+six total versions detailed below:
+
+#### `Inference Tests`
+
+| Inference Type | Client | Parallelization |
```

Comment:
Change Client to "Client Langauges"

Reply:
Recommend to do this for all tables

Reply:
It's not the inference that's colocated or standard, it's the database. (Same applies to all tables)

Reply:
Thank u so much this is so helpful to learn i did not realize